.github/workflows/publish.yml

```diff
@@ -23,8 +23,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Version
-        run: echo ${{ github.event.release.tag_name }} > VERSION
+        with:
+          fetch-depth: 0
```

Comment:
I just realized we should be providing the `tag_name` as the value for the `ref` option for the [checkout action](https://github.com/actions/checkout). It is currently possible to release a build based on a commit after the `tag_name`.

Correct me if I'm wrong.

Reply:
Hmmm yes I think so. Since checkout will always grab the latest version (afaik) it is possible that if another release commit gets merged between when the release was triggered and actually building the container. The result there would be the `VERSION` and container files being the updated version. The `VERSION` file would I think still be accurate to the actual contents, but the containers release tag could be misleading.

I assume there must be a way to check out the exact commit version, will investigate and get back to you. Pretty sure it would be no big deal to edit the dockerfile to use either the git commit history or the GH event tag